src/Content/Item.php

```diff
@@ -58,13 +59,15 @@ class Item
 	 *       ]
 	 *  ]
 	 */
-	public function determineCategoriesTerms(array $item)
+	public function determineCategoriesTerms(array $item, int $uid = 0)
 	{
 		$categories = [];
 		$folders = [];
 		$first = true;
 
-		foreach (Post\Category::getArrayByURIId($item['uri-id'], $item['uid'], Post\Category::CATEGORY) as $savedFolderName) {
+		$uid = ($item['uid'] != 0) ? $item['uid'] : $uid;
```

Comment:
This can be simplified to 
```suggestion
		$uid = $item['uid'] ?: $uid;
```